taxonomies/packaging_shapes.result.txt

```diff
@@ -61,16 +63,30 @@ nl:Fles
 pt:Garrafa, garrafas
 wikidata:en: Q80228
 
-en:Bottle cap, bottle top, cap, top, Wine cork, wine bottle cork, cork, bottle caps, bottle tops, caps, tops, wine corks, wine bottle corks, corks
-fr:Bouchon de bouteille, bouchon, bouchon de bouteille de vin, bouchons de bouteille, bouchons de bouteilles, bouchons
+en:Bottle cap, bottle top, cap, top, bottle caps, bottle tops, caps, tops
+fr:Bouchon de bouteille, bouchon, bouchons de bouteilles, bouchons
 hu:Kupak, borosdugó, dugó
+nl:Dop
 pt:Cápsula de garrafa, cápsulas de garrafa, tampa de garrafa, tampas de garrafa, rolha, rolhas, rolha de garrafa, rolhas de garrafa, rolha de cortiça, rolhas de cortiça, rolha de plástico, rolhas de plástico, carica, caricas, cápsula de rosca, cápsulas de rosca
 
+< en:Bottle cap
```

Comment:
So for this case, I had not put "capsules" under bottle caps, because the term is used for the Nespresso coffee capsules.